### DISCUSSION TITLE
Display Frontpage/Bsky votes/likes separately in post view

### DISCUSSION THREAD

**Discussion by erlend-sh** [likeandscribe/frontpage#168] [Ideas]
> When you post on Frontpage, we propose that a mirror post will also be created in your Bluesky account. When you comment on Frontpage, we propose that a mirror reply will be created in your Bluesky account.
> 
> Conversely, when you reply to one of these mirrored posts in Bluesky - we will show it as a reply in Frontpage.
> 
> Additionally, Bluesky likes will be translated to Frontpage votes and vice versa.

Yeh that sounds perfect to me 💯

Additionally though, I propose displaying these signal-sources separately when leaving feed-view to the singular post-view.

**Comment by WillCorrigan** [comment]
> > When you post on Frontpage, we propose that a mirror post will also be created in your Bluesky account. When you comment on Frontpage, we propose that a mirror reply will be created in your Bluesky account.
> > Conversely, when you reply to one of these mirrored posts in Bluesky - we will show it as a reply in Frontpage.
> > Additionally, Bluesky likes will be translated to Frontpage votes and vice versa.
> 
> Yeh that sounds perfect to me 💯
> 
> Additionally though, I propose displaying these signal-sources separately when leaving feed-view to the singular post-view.

Do you mean when you click a blueksy comment/post it will take you to a reddit-like view where it isolates the comment or that you just want to see other bsky replies and not frontpage ones?

Or just differentiate them and keep them together?

**Comment by erlend-sh** [comment]
Don’t think I’ll have a spare moment for the bit of mockup work required to avoid any confusion here, but it’s a very low priority proposal regardless. I’ll come back to it eventually if no one else does though!
